shared/javascripts/tooltip.js

```diff
@@ -204,14 +204,14 @@ var privlyTooltip = {
           // Fill only the first three columns with the coresponding values from glyphArray[]
           // The rest of two columns are simetrical to the first two
           if(j <= 2) {
-            if(glyphArray[i * 3 + j] == "true") {
+            if(glyphArray[i * 3 + j] === "true") {
```

Comment:
This is a common practice to have a simple and blanket "always use strict equals comparisons", but again, I can leave it out of this PR if you prefer.